ci-operator/step-registry/rosa/cluster/wait-ready/nodes/rosa-cluster-wait-ready-nodes-ref.yaml

```diff
@@ -17,6 +17,8 @@ ref:
   - name: READY_WAIT_TIMEOUT
     default: "10m"
     documentation: The environment variable to timeout for waiting before all nodes are ready.
-  
+  - name: MP_REPLICAS
+    default: ""
+    documentation: Count of machines for the machine pool (required when autoscaling is disabled). The replicas be a multiple of 3 for the multiAZ cluster.      
```

Comment:
What is the usage for MP_REPLICAS in the step `rosa-cluster-wait-ready-nodes`?

Reply:
my bad, it was later removed but forgot to update here. Thanks!